controlplane/src/core/services/Nodemailer.ts

```diff
@@ -8,11 +8,19 @@ interface OrganizationInviteBody {
   inviteLink: string;
 }
 
-export default class Postmark {
-  client: postmark.ServerClient;
+export default class Nodemailer {
+  client: Transporter;
 
   constructor(serverToken: string) {
-    this.client = new postmark.ServerClient(serverToken);
+    this.client = createTransport({
+      host: 'smtp.postmarkapp.com',
+      port: 587,
+      secure: false,
+      auth: {
+        user: serverToken,
```

Comment:
This should be configurable. On-premise some might really use a username / password combination. We will just use the token.

Reply:
Done